examples/notebooks/TextCNN.ipynb

```diff
@@ -168,149 +188,278 @@
-    "print ('Attributes of LABEL.vocab : ', [attr for attr in dir(LABEL.vocab) if '_' not in attr])\n",
-    "print ('First 5 values LABEL.vocab.itos : ', LABEL.vocab.itos) \n",
-    "print ('First 5 key, value pairs of LABEL.vocab.stoi : ', {key:value for key,value in list(LABEL.vocab.stoi.items())})\n",
-    "print ('Shape of LABEL.vocab.vectors : ', LABEL.vocab.vectors)"
+    "text_transform = lambda x: [vocab[token] for token in tokenizer(x)]\n",
+    "label_transform = lambda x: 1 if x == 'pos' else 0\n",
+    "\n",
+    "# Print out the output of text_transform\n",
+    "print(\"input to the text_transform:\", \"here is an example\")\n",
+    "print(\"output of the text_transform:\", text_transform(\"here is an example\"))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "ojXZisxLznkM"
+    "id": "xtZSEqjJQPxM"
+   },
+   "source": [
+    "For generating the data batches we will use `torch.utils.data.DataLoader`. You could customize the data batch by defining a function with the `collate_fn` argument in the DataLoader. Here, in the `collate_batch` func, we process the raw text data and add padding to dynamically match the longest sentence in a batch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "NHHIEfpRP4TV"
    },
+   "outputs": [],
    "source": [
-    "Now we must convert our split datasets into iterators, we'll take advantage of **torchtext.data.BucketIterator**! BucketIterator pads every element of a batch to the length of the longest element of the batch."
+    "from torch.utils.data import DataLoader\n",
+    "from torch.nn.utils.rnn import pad_sequence\n",
+    "\n",
+    "def collate_batch(batch):\n",
+    "    label_list, text_list = [], []\n",
+    "    for (_label, _text) in batch:\n",
+    "        label_list.append(label_transform(_label))\n",
+    "        processed_text = torch.tensor(text_transform(_text))\n",
+    "        text_list.append(processed_text)\n",
+    "    return torch.tensor(label_list), pad_sequence(text_list, padding_value=3.0)\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "id": "tM4yWV9YznkM"
+    "id": "3IQd3EVbQvTo"
    },
    "outputs": [],
    "source": [
-    "train_iterator, valid_iterator, test_iterator = legacy.data.BucketIterator.splits((train_data, valid_data, test_data), \n",
-    "                                                                           batch_size=32,\n",
-    "                                                                           device=device)"
+    "batch_size = 8  # A batch size of 8\n",
+    "\n",
+    "def create_iterators(batch_size=8):\n",
+    "    \"\"\"Heler function to create the iterators\"\"\"\n",
+    "    dataloaders = []\n",
+    "    for split in [train_list, validation_list, test_list]:\n",
+    "        bucket_dataloader = DataLoader(\n",
```

Comment:
We are not creating `bucket_dataloaders` anymore, as we don't use a `batch_sampler`, please edit this variable name.